pkgs/tools/text/vale/default.nix

```diff
@@ -1,10 +1,11 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGoModule, fetchFromGitHub }:
 
-buildGoPackage rec {
+buildGoModule rec {
```

Comment:
please revert this change, `buildGoPackage` is preferred when upstream vendorizes its dependencies

Reply:
Does it mean that documentation is inaccurate? This [section of Nixpkgs manual](https://nixos.org/nixpkgs/manual/#ssec-go-legacy) says exactly opposite:

    The function buildGoPackage builds legacy Go programs, not supporting Go modules.

Reply:
This is debated, `buildGoPackage` is recommended when upstream vendorizes its dependencies (`vendor/` directory). See: https://github.com/NixOS/nixpkgs/issues/84826